src/index.css

```
@@ -495,6 +498,17 @@ h1, h2, h3, h4, h5, h6 {
   justify-content: center;
   background: var(--gag-bg-blue);
   border-radius: 5px;
+  border-top: 1.5px solid var(--gag-highlight-blue);
+  border-bottom: 1.5px solid var(--gag-shadow-blue);
+  outline: 2px solid var(--gag-shadow-blue);
+  box-shadow: inset 0 0 6px var(--gag-highlight-blue),
+    0 2px 3px var(--mediumgrey);
+}
+
+.calc-gag-grid-cell:hover {
+  box-shadow: inset 0 -1px 9px 2px var(--gag-highlight-blue),
+    0 0px 0px var(--gag-shadow-blue);
+  border-bottom: 1px solid var(--gag-shadow-blue);
```

This, as well as 1 other suggestion, fixes odd styling after clicking a button (button gains `:focus`).
```suggestion
  box-shadow: inset 0 -1px 9px 2px var(--gag-highlight-blue),
    0 0px 0px var(--gag-shadow-blue) !important;
  border-bottom: 1px solid var(--gag-shadow-blue) !important;
```